userspace/libscap/engine/savefile/converter/converter.cpp

```diff
@@ -661,7 +661,14 @@ static conversion_result convert_event(std::unordered_map<uint64_t, safe_scap_ev
 	case C_ACTION_SKIP:
 		return CONVERSION_SKIP;
 
+	// TODO(ekoops): the current implementation of `C_ACTION_STORE` and `C_ACTION_STORE_AND_SKIP` is
+	//   identical. However, in the future, `C_ACTION_STORE` will return something to indicate that
+	//   the event should be dropped, while `C_ACTION_STORE_AND_SKIP` will return `CONVERSION_SKIP`
+	//   (or whatever is the name we choose to indicate that the event should proceed... Maybe we
+	//   can reuse `CONVERSION_COMPLETE`). Update the implementation once we are ready to change
+	//   the `C_ACTION_STORE` logic.
 	case C_ACTION_STORE:
+	case C_ACTION_STORE_AND_SKIP:
```

Comment:
nit: wouldn't make more sense to add `C_ACTION_STOR_AND_SKIP` later as the existing action does already what's needed at the moment?

Reply:
@ekoops for some reason I did not push this comment 😄 but it's no more relevant at this point

Reply:
I preferred to enforce it directly in the code. In this way, we dind't need to add a special notice in the scap converter table